packages/wxp-sdk-types/ui/ui-sdk.d.ts

```diff
@@ -252,6 +252,11 @@ export declare interface AppThemeChangeEventData {
     theme: UiTheme;
 }
 
+/**
+ * Asset collection Id type.
+ */
+export declare type AssetCollcetionId = `urn:aaid:sc:VA6C2:${string}`;
```

Comment:
There's a typo here. Let's fix this here and do a reverse fix in the original repo.